src/OpenTelemetry/Resources/Resource.cs

```diff
@@ -30,11 +30,15 @@ public class Resource
         /// Maximum length of the resource type name.
         /// </summary>
         private const int MaxResourceTypeNameLength = 255;
-        private readonly Dictionary<string, string> labelCollection;
 
```

Comment:
link to spec? https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-resource.md